builtin/lookup.go

```diff
@@ -1163,6 +1170,13 @@ option::copy includePatterns(variadic string pattern)
 # @return an option to copy files that don&#39;t match any pattern.
 option::copy excludePatterns(variadic string pattern)
 
+# Merges two input filesystems.
+#
+# @param input1 first filesystem to merge.
+# @param input2 second filesystem to merge.
+# @return a filesystem with input2 merged on top of input1
+fs merge(fs input1, fs input2)
```

Comment:
I wonder if this should be `fs merge(variadic fs input)` so we can `merge fs1 fs2 fs3 fs4`

Reply:
Made it variadic and added a codegen test.